packages/plugin-breadcrumbs-console/src/index.ts

```diff
@@ -45,4 +45,17 @@ function consoleBreadcrumbsPlugin(options?: { [key: string]: any }) {
   }
 }
 
+function serializeValue(value: any, method: string) {
+  if (typeof value === "string") {
+    return value
+  } else {
+    try {
+      return JSON.stringify(value)
+    } catch (error) {
+      console.error(`Could not serialize "console.${method}" to String.`, error)
```

Comment:
Will this call to `console.error` itself also be reported by the breadcrumbs console plugin? In that case, we would be sending two breadcrumbs, right? One with "Could not serialize ..." and one with "[Value could not be serialized]". Is that the desired behaviour?

Reply:
Yes it will be reported as a breadcrumb. I want it to log the error to the console. That also means, because of this plugin, it also gets added as a breadcrumb. We could ignore it, but this is what the plugin does. It adds these kinds of messages as breadcrumbs, even if they're from the package itself. I think users would want to know this error was logged.

The only thing I was concerned with was causing an infinite loop of errors having a circular structure as well, but that doesn't look like it's happening here.